projects/ion/src/lib/card/card.component.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { IonIconComponent } from '../icon/icon.component';
+import { CardEvent, CardIonComponent, IonCard } from './card.component';
+import { ButtonComponent } from '../button/button.component';
+import { TestBed } from '@angular/core/testing';
+import { CommonModule } from '@angular/common';
+import {
+  AfterViewInit,
+  ChangeDetectorRef,
+  Component,
+  NgModule,
+} from '@angular/core';
+import { fireEvent, screen } from '@testing-library/angular';
+
+let renderFooter = false;
+
+@Component({
+  template: `<ion-button
+    [label]="label"
+    [type]="'primary'"
+    data-testid="buttonBody"
+  ></ion-button>`,
+})
+class ButtonTestComponent {
+  public label = 'Botão no body';
+}
+
+@Component({
+  template: `<h1 data-testid="footerTest">Aqui é o footer</h1>`,
+})
+class FooterTestComponent {}
+
+@Component({
+  template: `<ion-card
+    [configuration]="cardConfi"
+    (events)="cardEvents($event)"
+  ></ion-card>`,
+})
+class CardTestComponent implements AfterViewInit {
+  constructor(private cdr: ChangeDetectorRef) {}
+  ngAfterViewInit(): void {
+    this.cdr.detectChanges();
+  }
+  public cardConfi: IonCard = {
+    body: ButtonTestComponent,
+    header: { title: 'opa' },
+    footer: renderFooter
+      ? { bodyFooter: FooterTestComponent }
+      : { buttons: { primary: { label: 'iea' } } },
+  };
+
+  public cardEvents(event: CardEvent) {
+    this.cardConfi.header.title = `Opa, eu fui clicado evento: ${event.buttonAction}`;
+  }
+}
+
+@NgModule({
+  imports: [CommonModule],
+  declarations: [
+    CardTestComponent,
+    CardIonComponent,
+    ButtonComponent,
+    IonIconComponent,
+    ButtonTestComponent,
+    FooterTestComponent,
+  ],
+  entryComponents: [
+    CardTestComponent,
+    ButtonTestComponent,
+    FooterTestComponent,
+  ],
+})
+class TestModule {}
+
+describe('CardComponent', () => {
+  beforeEach(async () => {
+    TestBed.configureTestingModule({
+      imports: [TestModule],
+    }).compileComponents();
+  });
```

Comment:
```suggestion
    TestBed.configureTestingModule({
      imports: [TestModule],
    }).compileComponents();
  });
```

add sut and fixture.detectChanges(); here.